apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -86,18 +87,18 @@ const Profile: React.FC = () => {
 			{isLoading && <Loader />}
 			{user && (
 				<div className={styles["page-container"]}>
-					<h4 className={styles["title"]}>Profile</h4>
-					<div className={styles["content-container"]}>
+					<ProfileSection title="Profile">
```

Comment:
'Profile' is title of the whole page, not of the section. I think it should have bigger font-size than other titles on this page.